app/scripts/controllers/monitoring.js

```diff
@@ -29,7 +29,8 @@ angular.module('openshiftConsole')
     $scope.renderOptions.showEventsSidebar = true;
     $scope.renderOptions.collapseEventsSidebar = localStorage.getItem('monitoring.eventsidebar.collapsed') === 'true';
 
-
+    var limitWatches = $filter('isIE')();
```

Comment:
Wonder if we shouldn't set `LIMIT_WATCHES` as a constant once in app.js. Then if we ever need to change it again, we're always consistent.